### ISSUE TITLE
[Bug]: API call to delete a Edge Rule for a given Pull Zone sends the wrong data for the edge rule

### ISSUE THREAD

**Issue by ricardoee** [ToshY/BunnyNet-PHP#134]
### Package version(s) affected

4.4.0

### Description

Great library apart from this one bug.  I noted that the API call to correctly delete an Edge Rule from a Pull Zone actually needs a `Guid` for the Edge Rule, not a integer id.

For example.  The library produces the following request. 

```
Nyholm\Psr7\Request {#333
  -headers: array:2 [
    "Host" => array:1 [
      0 => "api.bunny.net"
    ]
    "AccessKey" => array:1 [
      0 => "REMOVED"
    ]
  ]
  -headerNames: array:2 [
    "accesskey" => "AccessKey"
    "host" => "Host"
  ]
  -protocol: "1.1"
  -stream: null
  -method: "DELETE"
  -requestTarget: null
  -uri: 
Nyholm\Psr7
\
Uri {#344
    -scheme: "https"
    -userInfo: ""
    -host: "api.bunny.net"
    -port: null
    -path: "/pullzone/2486260/edgerules/457"
    -query: ""
    -fragment: ""
  }
}
```

The actual path uses an integer id for referencing the Edge Rule, but it's actually a `Guid`.

### How to reproduce

My actual code

```
    private function deletePullZoneEdgeRule(PullZoneDto $pullZone, string $guid): void
    {
        try {
            $this->client->deleteEdgeRule(
                pullZoneId: $pullZone->pullZoneId,
                edgeRuleId: $guid,
            );
        } catch (
            ClientExceptionInterface|
            BunnyClientResponseException|
            JSONException $e
        ) {
            $pullZone->success = false;
            $pullZone->message[] = \sprintf(
                'Could not delete Edge Rule "%s" for Pull Zone "%s" - "%s"',
                $guid,
                $pullZone->pullZoneId,
                $pullZone->pullZoneName
            );
            $pullZone->message[] = $e->getMessage();
        )
    }
```

### Possible Solution

This is my working code that sends the following parameters:

```
$pullZone->pullZoneId = 2486260;
$guid = '0cc6f8b7-45bf-43dd-9321-634ea5acd8df';
```

#### Symfony
```
$client = (new HttpClient())
            ->createForBaseUri('https://api.bunny.net/', [
                'headers' => [
                    'AccessKey' => $this->apiKey,
                    'Content-Type' => 'application/json',
                    'Accept' => 'application/json',
                ]
            ])
            ->request('DELETE', \sprintf('pullzone/%u/edgerules/%s', $pullZone->pullZoneId, $guid));
```

#### curl
```
curl -X DELETE -v -s --location 'https://api.bunny.net/pullzone/2486260/edgerules/0cc6f8b7-45bf-43dd-9321-634ea5acd8df' \
--header 'AccessKey: REMOVED' \
-H 'Content-Type: application/json' \
-H 'Accept: application/json'
```

### Additional Context

Can you let me know what the eta to fix this would be?

**Comment by ToshY** [comment]
Hey @ricardoee 👋 

Thanks for reporting it! You're right it should be the guid/uuid of the edge rule.

> Can you let me know what the eta to fix this would be?

Somewhere in the next couple of hours. I'll mention you when a new release is available.

**Comment by ToshY** [comment]
@ricardoee This has been fixed in [4.4.1](https://github.com/ToshY/BunnyNet-PHP/releases/tag/4.4.1)

**Comment by ricardoee** [comment]
@ToshY Thank you, will update my dependencies and test.
